main.nf

```diff
@@ -20,7 +20,7 @@ params.help = null
 params.debug = null
 download_vcf = null
 params.finemap = true
-params.species = "c_elegans"
+// params.species = "c_elegans"
```

Comment:
if appropriate, add a comment here explaining where the value is being injected.

Reply:
I've removed this line: 810113e878f039598ff7d966414916c65f86111a